eftest/src/eftest/runner.clj

```diff
@@ -5,10 +5,24 @@
             [clojure.tools.namespace.find :as find]
             [eftest.report :as report]
             [eftest.report.progress :as progress]
-            [eftest.output-capture :as capture]))
+            [eftest.output-capture :as capture])
+  (:import [java.util.concurrent Executors ExecutorService]))
 
 (defmethod test/report :begin-test-run [_])
 
+(defn make-threadpool
+  ([] (make-threadpool (.availableProcessors (Runtime/getRuntime))))
+  ([n]
+   (Executors/newFixedThreadPool n)))
+
+(defn tp-pmap
```

Comment:
Maybe this would be better named `executor-pmap` and written with a threading macro for clarity:

```clojure
(defn- executor-pmap  [^ExecutorService executor f coll]
  (->> (for [e coll :let [^Callable cb (cast Callable (bound-fn [] (f e)))]]
         (.submit executor cb))
       (doall)
       (map deref)))
```